src/components/SshKeys/AddSshKey.js

```diff
@@ -82,7 +82,7 @@ const AddSshKey = ({me: { id, email }}) => {
                   className="addSshKeyInput"
                   type="text"
                   onChange={handleChange}
-                  value={values.sshKey}
+                  value={values.sshKey.includes('\n') ? values.sshKey.trim() : values.sshKey}
```

Comment:
So I'm not sure of this logic.

Our test for newlines is "look and see if there is a \n in the string", but our solution if we find one is "trim the edges of the string". These two are out of whack.

To demo:
```
if("test 1\n".trim().includes("\n")) {
  console.log("test 1 has a newline");
}

if("test\n 2".trim().includes("\n")) {
   console.log("test 2 has a newline");
}
```
So I guess my question is - what are we testing for? **Any** newlines, or just newlines at the beginning and end of the string?
If it's the latter, I'd say lets just trim the string every time, rather than run the (inaccurate) check.
If we're concerned about newlines in the middle of strings, then we need to do a global substitution or something.

As a second step, maybe a better strategy would be to actually have feedback in the UI saying that the format of the key is invalid (and provide reasons why).

Reply:
Thanks! As discussed I've removed that validation and created a new [issue](https://github.com/uselagoon/lagoon-ui/issues/64) to clean up the validation and add feedback in the UI.